samplecode/singularity/concat_csv.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import csv
+import pathlib
+import typing as ty
+import unittest
+
+PARSER = argparse.ArgumentParser()
```

Comment:
Unless you have strong feelings about it, I prefer wrapping the argument parser in a `parse_args()` function. It doesn't make much difference for a trivial case like this, but when you have to calculate some default values or add other logic, it's nice to avoid doing all that at the module scope.

Here's an [example] from MiCall.

[example]: https://github.com/cfe-lab/MiCall/blob/v7.12.1/micall/utils/sort_sam.py#L8-L12